### PR TITLE
ADMINUI-2339 Should not allow user to enter an IPv4 address for DNS w…

### PR DESCRIPTION
**Pull request by andyYatskov**
…hen Subnet is IPv6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,10 +35,10 @@
     "reactify": "1.0.0",
     "restify": "4.3.1",
     "sdc-clients": "10.2.1",
-    "ufds": "1.3.0",
     "superagent": "0.21.0",
     "through": "2.3.4",
     "trace-event": "1.3.0",
+    "ufds": "1.1.3",
```
Comment (dan-luk) [inline package.json]: why roll it back??

**Review by dan-luk — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (andyYatskov) [reply package.json]: I used npm i --save and this command makes the alphabetical sorting in the dependencies.

Comment (andyYatskov) [reply package.json]: this happens after npm i -- save and rebase

**Review by dan-luk — APPROVED** [review]
🔢

**Pull request closed without merging**